### DISCUSSION TITLE
SourceTable of the DrillDownPageID / LookupPageID must match the Table ID

### DISCUSSION THREAD

**Discussion by fvet** [StefanMaron/BusinessCentral.LinterCop#39] [New Rule]
We sometimes make a full copy of table from Business Central to new tables, copy from an existing table to a buffer / temporary / staging / import table, ... When copying these source tables, sometimes the DrillDownPageID and LookupPageID are forgotten to be updated on the new target table and still reference to the 'old' source table, which is wrong. DrillDown and Lookup will not work properly on the newly created table, since the SourceTable of the pages referenced by DrillDownPageID / LookupPageID does not match the ID of the parent table.

Possible overlap with Future AL CodeCop... 
More info > https://github.com/microsoft/AL/issues/6099

**Comment by Arthurvdv** [comment]
This is now handled in the [LC0039](https://github.com/StefanMaron/BusinessCentral.LinterCop/wiki/LC0039) `The given argument has a different type from the one expected.` rule.
Should we close off this discussion?
